### PR TITLE
TGGjets_leptonDecays_4f_ckm_LO card

### PR DESCRIPTION
**Pull request by ckmackay**
TGGjets_leptonDecays_4f_ckm_LO card

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/TGGjets_leptonDecays_4f_ckm_LO/TGGJets_leptonDecays_4f_ckm_LO_madspin_card.dat [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+define ell+ = e+ mu+ ta+
+define ell- = e- mu- ta-
+# specify the decay for the final state particles
+decay t > w+ b, w+ > ell+ vl
+decay t~ > w- b~, w- > ell- vl~
+decay w+ > ell+ vl
+decay w- > ell- vl~
+
+#define ell+ = e+ mu+ ta+
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/TGGjets_leptonDecays_4f_ckm_LO/TGGJets_leptonDecays_4f_ckm_LO_madspin_card.dat]: can you please cleanup the datacards?

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/TGGjets_leptonDecays_4f_ckm_LO/TGGJets_leptonDecays_4f_ckm_LO_run_card.dat [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+     0.0     = polbeam1 ! beam polarization for beam 1
+     0.0     = polbeam2 ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+     lhapdf    = pdlabel     ! PDF set                                     
+     292200    = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/TGGjets_leptonDecays_4f_ckm_LO/TGGJets_leptonDecays_4f_ckm_LO_run_card.dat]: PDF is wrong, likely you have to use 263400, cfr https://lhapdf.hepforge.org/pdfsets.html

**Comment by perrozzi** [comment]
ping

**Comment by ckmackay** [comment]
HI,

Sorry, I've been away. I've checked with the person who wrote the cards and yes the pdf set is incorrect. I'll change it now and tidy up the others and resubmit.

Many thanks, Kate
________________________________

**Pull request closed without merging**